CONTRIBUTING.md

```diff
@@ -737,6 +737,39 @@ For example:
 > [!NOTE]
 > To upload images, use GitHub Desktop. See [Making a larger contribution](#making-a-larger-contribution).
 
+### Video
+
+To embed a video, you have a couple of options:
+
+- Use the following syntax, which will only work in **docfx**.
```

Comment:
Since our contributing file is meant for DocFX only, I'd simplify this as follows: 
```suggestion
- Use the following syntax:
```